graphs/digraph.py

```diff
@@ -1,3 +1,4 @@
+import os
 from graph import graph
 from copy import deepcopy
```

Comment:
standard import \from copy import deepcopy\ should be placed before \from graph import graph\